spec/components/s3_helper_spec.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
-require "s3_helper"
 require "rails_helper"
+require "s3_helper"
```

Comment:
When running this test singly I got an error saying `.minutes` wasn't an integer method, so rails gubbins hadn't been loaded yet :)

https://github.com/discourse/discourse/blob/6071dc81c72c95f8e2b1c66991dae8399a79798c/lib/s3_helper.rb#L18-L18